### ISSUE TITLE
Broken internal links on the WCAG 2.2 Understanding and Techniques pages

### ISSUE THREAD

**Issue by TestPartners** [w3c/wcag#4962]
In response to [Patrick's suggestion](https://github.com/w3c/wcag/issues/4961#issuecomment-3972232103), I have run an automated link checker on the Understanding and Techniques pages for WCAG 2.2. The broken internal links are listed below. I opened a [separate issue for the broken external links](https://github.com/w3c/wcag/issues/4964).

https://www.w3.org/WAI/WCAG22/Techniques/css/focus-not-obscured-minimum
This is linked from the "Focus Appearance (Minimum)" link on https://www.w3.org/WAI/WCAG22/Techniques/css/C41

https://www.w3.org/WAI/WCAG22/Techniques/css/technique-using-css-to-ensure-targets-are-at-least-44-by-44-css-pixels.html
This is linked from the "CXXXXXX" link on https://www.w3.org/WAI/WCAG22/working-examples/css-44px-target-size/

https://www.w3.org/WAI/WCAG22/Techniques/failures/motion-actuation
This is linked from the "Motion Actuation Success Criterion" link under the Description heading on https://www.w3.org/WAI/WCAG22/Techniques/failures/F106

https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interaction
This is linked from the "2.3.3 Animation from Interaction" link under the Intent heading on https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html
Both the visible text and the URL should refer to "interactions" rather than "interaction".

The following are all broken links to CSS files:

https://www.w3.org/WAI/WCAG22/Techniques/css/TBC
This is linked from the "Allowing for Reflow with Long URLs and Strings of Text" link on https://www.w3.org/WAI/WCAG22/working-examples/css-reflow-url/

https://www.w3.org/WAI/WCAG22/working-examples/reflow-indenting/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/reflow-indenting/

https://www.w3.org/WAI/WCAG22/working-examples/reflow-nested-lists/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/reflow-nested-lists/

https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-shoppingcart/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-shoppingcart/

https://www.w3.org/WAI/WCAG22/working-examples/failure-table-headers-incorrect-association/style/oturn.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/failure-table-headers-incorrect-association/

https://www.w3.org/WAI/WCAG22/working-examples/failure-status-message/sources.css
This is linked from a `<link>` tag on 
https://www.w3.org/WAI/WCAG22/working-examples/failure-status-message/

https://www.w3.org/WAI/WCAG22/working-examples/aria-role-log/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/aria-role-log/chatlog.html

https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-searchresults/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-searchresults/

https://www.w3.org/WAI/WCAG22/css/sources.css
This is linked from a `<link>` tag on https://www.w3.org/WAI/WCAG22/working-examples/reflow-carousel-panel-horizontal-scroll/

**Comment by fstrr** [comment]
1. link on https://www.w3.org/WAI/WCAG22/Techniques/css/C41 : link doesn't exist on that page
2. https://www.w3.org/WAI/WCAG22/working-examples/css-44px-target-size/ : already fixed
3. https://www.w3.org/WAI/WCAG22/Techniques/failures/motion-actuation : fixed
4. https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interaction : link ready fixed, typo addressed in #5290
5. https://www.w3.org/WAI/WCAG22/working-examples/css-reflow-url/ : already fixed
6. https://www.w3.org/WAI/WCAG22/working-examples/reflow-indenting/ : already fixed
7. https://www.w3.org/WAI/WCAG22/working-examples/reflow-nested-lists/ : already fixed
8. https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-shoppingcart/ : already fixed
9. https://www.w3.org/WAI/WCAG22/working-examples/failure-table-headers-incorrect-association/ : already fixed
10. https://www.w3.org/WAI/WCAG22/working-examples/failure-status-message : already fixed
11. https://www.w3.org/WAI/WCAG22/working-examples/aria-role-log/chatlog.html : already fixed
12. https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-searchresults : already fixed
13. https://www.w3.org/WAI/WCAG22/working-examples/reflow-carousel-panel-horizontal-scroll : already fixed
